spring-boot-project/spring-boot/src/main/java/org/springframework/boot/availability/ApplicationAvailabilityBean.java

```diff
@@ -84,7 +84,7 @@ private <S extends AvailabilityState> Object getLogMessage(Class<S> type, Availa
 		StringBuilder message = new StringBuilder(
 				"Application availability state " + type.getSimpleName() + " changed");
 		message.append((lastChangeEvent != null) ? " from " + lastChangeEvent.getState() : "");
-		message.append(" to " + event.getState());
+		message.append(" to ").append(event.getState());
```

Comment:
I'm afraid I don't think we should do this one either. There's not going to be much of a performance impact.